ledger/metrics/src/main/scala/com/daml/metrics/Timed.scala

```diff
@@ -57,31 +53,24 @@ object Timed {
   }
 
   def future[T](timer: Timer, future: => Future[T]): Future[T] = {
-    this.future[T](timer.metric, future)
-  }
-
-  def future[T](timer: codahale.Timer, future: => Future[T]): Future[T] = {
```

Comment:
this function must be exported for canton

Reply:
It depends on `Dropwizard`, canton should not use dropwizard metrics directly.